crux-bench/src/crux/bench.clj

```diff
@@ -27,8 +27,10 @@
 (def ^:dynamic ^:private *bench-dimensions* {})
 (def ^:dynamic ^:private *!bench-results*)
 
+(def !crux-node-type (atom ""))
+
 (defn with-dimensions* [dims f]
-  (binding [*bench-dimensions* (merge *bench-dimensions* dims)]
+  (binding [*bench-dimensions* (merge *bench-dimensions* dims {:crux-node-type @!crux-node-type})]
```

Comment:
this will bind the crux-node-type every time with-dimensions is called - you could call with-dimensions from your own function?